editor/editor_node.cpp

```diff
@@ -47,7 +47,6 @@
 #include "core/version.h"
 #include "main/main.h"
 #include "scene/gui/color_picker.h"
-#include "scene/gui/dialogs.h"
```

Comment:
Did you remove it because it's redundant due to indirect inclusion from FileDialog? I think we prefer explicitness with includes, as otherwise some weird dependency issues crop up.

Reply:
It was moved it the header, and I forgot to put it back. Fixed.